docs/reference/commandline/network_inspect.md

```diff
@@ -124,6 +123,63 @@ $ docker network inspect simple-network
 ]
 ```
 
+For swarm mode overlay networks `network inspect` also shows the IP and node name of 
+peers. Peers are the nodes in the swarm cluster which has atleast one task attached 
+to the network. Node name is of the format `hostname-unique ID`.
```

Comment:
Perhaps use: `<hostname>-<unique ID>` (I read it as "hostname-unique" space "ID")